SU2_CFD/src/iteration/CFluidIteration.cpp

```diff
@@ -232,8 +232,11 @@ bool CFluidIteration::Monitor(COutput* output, CIntegration**** integration, CGe
                            config, TurbomachineryStagePerformance, TurbomachineryPerformance, val_iZone, config[val_iZone]->GetTimeIter(), config[val_iZone]->GetOuterIter(),
                            config[val_iZone]->GetInnerIter(), val_iInst);
     }
-
-    TurboMonitor(geometry, config, config[val_iZone]->GetInnerIter(), val_iZone);
+    /*--- Update ramps, grid first then outlet boundary ---*/
+    if (config[val_iZone]->GetRampMotionFrame())
+      TurboRamp(geometry, config, config[val_iZone]->GetInnerIter(), val_iZone, TURBO_RAMP_TYPE::GRID);
+    if (config[val_iZone]->GetRampOutflow())
+      TurboRamp(geometry, config, config[val_iZone]->GetInnerIter(), val_iZone, TURBO_RAMP_TYPE::BOUNDARY);
```

Comment:
I think the ramp function for the outflow should be placed outside the scope of `config[ZONE_0]->GetBoolTurbomachinery()`, since Riemann bc can also be used for non-turbo simulations.